### DISCUSSION TITLE
essay/2024/07/08/steady-blogging

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#217] [Comments]
# essay/2024/07/08/steady-blogging

개발자 블로그를 큐레이션 한지 8년째.

https://jeho.page/essay/2024/07/08/steady-blogging.html

**Comment by natz92** [comment]
개발자 이야기가 완결이 나지 않길 바라야겠네요!

**Reply by BenjaminKim** [reply]
고맙습니다. 저도 오랫동안 더 하고 싶어요. 열정이 계속 남아있어야 할텐데요. 😄
